autoextract_spiders/spiders/autoextract_spider.py

```diff
@@ -149,7 +156,8 @@ def make_extract_request(self, url, meta=None, check_page_type=True):
         if is_blacklisted_url(url):
             self.crawler.stats.inc_value('error/blacklisted_url')
             return
-
+        meta = meta or {}
+        meta['cf_store'] = True
```

Comment:
All AutoExtract requests always come to Frontera/HCF - it'll allow us to split producer/consumer behavior if it's needed.